common/src/main/java/com/wynntils/features/chat/ChatMentionFeature.java

```diff
@@ -63,7 +65,13 @@ public void onChat(ChatMessageReceivedEvent e) {
 
         StyledText styledText = StyledText.fromComponent(message);
 
-        StyledText modified = styledText.iterate((part, changes) -> {
+        StyledText modified = styledText.iterateBackwards((part, changes) -> {
```

Comment:
Ok, but now you are only marking the *last* mention of your name. I thought just marking the first, instead of all, was borderline buggy. But this is just weird. Please mark all instances, basically start at the end and iterate backwards and mark all instances until you reach the header.

Reply:
Done.